### ISSUE TITLE
False positive #1? 

### ISSUE THREAD

**Issue by geoffrk00** [microsoft/accessibility-insights-service#2259]
https://webcompliance.trafficmanager.net/ScanResultsApi/GetAccessibilityScanConsolidatedReport?siteId=8172d1b3-b36c-e711-80c2-0004ffc0fa4b

WCP is showing an error on 
https://learn.microsoft.com/en-us/
duplicate-id-aria: Ensures every id attribute value used in ARIA and in labels is unique

<div class="dropdown-menu" id="theme-menu" role="menu">

This ID is not duplicated 
Running AI4W locally on this page does not show this error

**Comment by ferBonnin** [comment]
rerouting this to the right repo.
cc @lamaks for visibility

**Comment by jlperkins** [comment]
@geoffrk00 I took a look at this issue again, and actually I am noticing that the failure looks legitimate; it seems it is getting caused by the duplicate footer at the very bottom of the page. Here is a screenshot of the double footer that I am seeing: 
![image](https://user-images.githubusercontent.com/33770444/193941495-513e5d05-b0b2-410c-8891-5dd4f960518a.png)

I did notice that refreshing the page seems to resolve the double footer issue. However, I can consistently reproduce the double footer on the first time I navigate to the site from a new tab. Let me know if you also see this, and if you have any more questions for us.

**Comment by geoffrk00** [comment]
Fantastic Catch!!!!
Thank you so much for finding this and providing the screenshot. 

We were able to find a way to reproduce this. It was complicated, but this resulted from a combination of HTTP chunked delivery combined with an experiment script. The result was that the footer wound up getting duplicated. 

We have found a simple way to prevent this by ensuring that the experiment script waits until the dom is completely loaded before beginning it's experiment. 

We are in the process of deploying this script now. We will rerun the accessibility scan once we have this deployed. 

In any outcome, it is clear that this is not a problem with the scanner. 
This issue can be resolved. 

Again, than you so much for spotting this.
